Eduria/Eduria/Controllers/ExamController.cs

```diff
@@ -35,9 +46,98 @@ public ExamController(ExamService examService, QuestionService questionService,
         public IActionResult Show(int id=1)
         {
             //return View(GetExamDataById(id));
```

Comment:
Gaat deze method in de toekomst nog gebruikt worden?
Anders weghalen

Reply:
GetExamDataById is nu verwijderd